variables.tf

```diff
@@ -164,3 +165,25 @@ variable "random_suffix" {
   type        = bool
   default     = true
 }
+
+variable "encrypt_gcs_bucket_tfstate" {
+  description = "Encrypt bucket used for storing terraform state files in seed project."
+  type        = bool
+  default     = false
+}
+
+variable "key_protection_level" {
+  type        = string
+  description = "The protection level to use when creating a version based on this template. Default value: \"SOFTWARE\" Possible values: [\"SOFTWARE\", \"HSM\"]"
+  default     = "SOFTWARE"
+}
+
+variable "key_rotation_period" {
+  type    = string
+  default = null
+}
+
+variable "prevent_destroy" {
```

Comment:
nit: If this only applies to KMS, it should probably be prefixed with `kms_`

Reply:
done